src/map_loader.cpp

```diff
@@ -63,36 +62,13 @@ namespace
 // Case-unsensitive strings equality-comparision
 static bool StringEquals( const char* const s0, const char* const s1 )
 {
-	unsigned int i= 0;
-	while( s0[i] != '\0' && s1[i] != '\0' )
-	{
-		if( std::tolower( s0[i] ) != std::tolower( s1[i] ) )
-			return false;
-		i++;
-	}
-
-	return std::tolower( s0[i] ) == std::tolower( s1[i] );
+	return ( ::strcasecmp( s0, s1 ) == 0 );
 }
 
 // Case-unsensitive substring search
 static const char* GetSubstring( const char* const search_where, const char* const search_what )
 {
-	const char* str= search_where;
-	while( *str != '\0' )
-	{
-		unsigned int i= 0u;
-
-		while( str[i] != '\0' && search_what[i] != '\0' &&
-				std::tolower( str[i] ) == std::tolower( search_what [i] ) )
-			i++;
-
-		if( search_what [i] == '\0' )
-			return str;
-
-		str++;
-	}
-
-	return nullptr;
+	return ( ::strcasestr( search_where, search_what ) );
```

Comment:
This function i can't see in the [standard](https://en.cppreference.com/w/cpp/header/cstring).

Reply:
The function is available with #define _GNU_SOURCE
I've added _MSC_VER windows defines too.